oggm/core/models/massbalance.py

```diff
@@ -201,7 +201,7 @@ def __init__(self, gdir, mu_star=None, bias=None, prcp_fac=None,
         self.temp_bias = 0.
 
         # Read file
-        fpath = gdir.get_filepath(filename, filesuffix=filesuffix)
+        fpath = gdir.get_filepath(filename, input_filesuffix=input_filesuffix)
```

Comment:
No need for a new keyword here: simply write ``fpath = gdir.get_filepath(filename, filesuffix=input_filesuffix)``